lib/uffizzi/cli/dev.rb

```diff
@@ -55,16 +56,34 @@ def run(command, command_args = {})
     end
 
     def handle_start_command(command_args)
-      config_path = command_args[:config_path]
       DevService.check_skaffold_existence
-      DevService.check_no_running_process!
+
+      config_path = command_args[:config_path]
       DevService.check_skaffold_config_existence(config_path)
+      dev_cluster = DevService.account_user_project_dev_cluster(server, account_id, project_slug)
+
+      if try_to_start_second_dev_env?(dev_cluster)
+        msg = 'Dev cluster already exists. ' \
+              "Before you create a new dev cluster, you must delete the existing one: $ uffizzi cluster delete #{dev_cluster[:name]}"
+        Uffizzi.ui.say_error_and_exit(msg)
+      end
+
+      if dev_cluster_already_deleted?(dev_cluster)
+        DevService.clear_after_delete(dev_environment[:cluster_id], dev_environment[:encoded_kubeconfig])
+        answer = Uffizzi.prompt.yes?('The previous development environment has been deleted. Do you want to start a new one?')
 
-      if dev_environment.empty?
+        return unless answer
+      end
+
+      DevService.check_no_running_process!
+
+      unless DevService.dev_environment_exist?
```

Comment:
Is this check same as `try_to_start_second_dev_env?`?

Reply:
It checks existence the dev env in local config